cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -293,12 +294,12 @@ data:
   Corefile: |
     .:53 {
         errors
-        log
         health
         kubernetes {{ .DNSDomain }} {{ .ServiceCIDR }} {
            pods insecure
+           upstream /etc/resolv.conf
```

Comment:
What benefits do we get from this?

Reply:
@chrisohaver wrote in https://github.com/coredns/deployment/pull/36#issue-283057544:
>We should include upstream by default, so externalName services can resolve.